app/main.py

```diff
@@ -14,0 +1,13 @@
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+
+    Actor.objects.create(first_name="Emma", last_name="Watson")
+    Actor.objects.create(first_name="Daniel", last_name="Radcliffe")
+    print(Actor.objects.all())
+    Actor.objects.update(2, "Daniel", "Radcliffe")
+    print(Actor.objects.all())
+    Actor.objects.delete(1)
+    print(Actor.objects.all())
```

Comment:
Consider closing the database connection after all operations are completed to prevent potential resource leaks.

Reply:
how can i close it?